pages/linux/microcom.md

```diff
@@ -0,0 +1,11 @@
+# microcom
+
+> A minimalistic terminal program.
+
+- Open a serial port using the specified baudrate:
```

Comment:
Perhaps `baudrate` could be `baud rate`?

Reply:
Not a question of perhaps, it _must_ be :laughing:

Reply:
Yes, you're right: The correct term is ``baud rate``, two words. Fixed.